quetz/tests/test_cli.py

```diff
@@ -13,40 +13,46 @@ def user_group():
 
 @pytest.fixture
 def config_extra(user_group):
-    return f"""[users]
-    {user_group} = ["bartosz"]
-    """
+    if user_group is None:
+        return ""
+    else:
+        return f"""[users]
+                {user_group} = ["bartosz"]
+                """
 
 
-@pytest.mark.parametrize(
-    "user_group,expected_role",
-    [("admins", "owner"), ("maintainers", "maintainer"), ("members", "member")],
-)
-def test_init_db(db, config, config_dir, user_group, expected_role):
+def get_user(db, config_dir):
+
```

Comment:
You Need to remove this empty line to satisfy linter
```suggestion
```

Reply:
Thanks! I've just made another commit to move the init_db and remove this empty line :)

Reply:
Cool, thanks!